grails-app/services/org/pih/warehouse/requisition/RequisitionService.groovy

```diff
@@ -745,5 +745,40 @@ class RequisitionService {
         }
     }
 
+    def getRequisitionCountInCurrentFiscalYear(Location location) {
+        def fiscalYearConfig = grailsApplication.config.openboxes.dashboard.yearTypes.fiscalYear
```

Comment:
What are the possible values (and default value) for this property? Has it already been added to Config.groovy?

Reply:
https://github.com/openboxes/openboxes/pull/3150/files all was added in this PR